react/react2/week1-exercise/app/components/NavBar.jsx

```diff
@@ -0,0 +1,127 @@
+"use client";
+import {
+  AppBar,
+  Toolbar,
+  IconButton,
+  Typography,
+  Stack,
+  Button,
+  useMediaQuery,
+  ListItem,
+  List,
+  Drawer,
+  ListItemButton,
+} from "@mui/material";
+import MenuIcon from "@mui/icons-material/Menu";
+import { useState } from "react";
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+
+function NavBar() {
+  const [drawerOpen, setDrawerOpen] = useState(false);
+  const isMobile = useMediaQuery("(max-width:600px)");
+  const pathname = usePathname();
+
+  const toggleDrawer = () => {
+    setDrawerOpen(!drawerOpen);
+  };
+
+  const getButtonClasses = (path) => {
+    return pathname === path ? "font-bold text-base" : "";
+  };
+
+  const drawer = (
+    <List>
+      <ListItem>
+        <ListItemButton
+          href="/astronomy-picture"
+          className={getButtonClasses("/astronomy-picture")}
+        >
+          Astronomy Picture
+        </ListItemButton>
+      </ListItem>
+      <ListItem>
+        <ListItemButton
+          href="/rover-photos"
+          className={getButtonClasses("/rover-photos")}
```

Comment:
This is a cool implementation but why not just create an array to .map over instead of hardcode and repeat this?